injectproxy/routes.go

```diff
@@ -405,3 +420,27 @@ func matchersToString(ms ...*labels.Matcher) string {
 	}
 	return fmt.Sprintf("{%v}", strings.Join(el, ","))
 }
+
+type QueryParseError struct {
```

Comment:
Here too re: exporting
```suggestion
type queryParseError struct {
```

Reply:
Done